packages/flutter/test/widgets/listener_deprecated_test.dart

```diff
@@ -107,8 +107,8 @@ void main() {
       final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
       await gesture.addPointer(location: Offset.zero);
       addTearDown(gesture.removePointer);
-      await gesture.moveTo(const Offset(400.0, 300.0));
       await tester.pump();
+      await gesture.moveTo(const Offset(400.0, 300.0));
```

Comment:
When you say:

> There must be a pump right after adding the pointer, which makes the MouseRegion repaint and add the annotation layer.

Does this mean that this PR changes the timing of the annotation layer addition?  What happens if you don't change the order of pumps and gestures in these tests: do they now fail after this PR, or were they just incorrectly succeeding?

Reply:
If I don't change it, the tests will now fail. But that will not happen in a real application, because a `RenderMouseRegion` that has any callbacks always triggers a rerender when the first mouse is connected.

Reply:
OK, as long as it can't change the application behavior.